### PR TITLE
[RSDK-2605] Document expectations of color field in PCDs

### PR DESCRIPTION
**Pull request by dmhilly**
This PR adds a short comment documenting how Viam interprets the color field in a PCD.

### CODE REVIEW

File: proto/viam/service/slam/v1/slam.proto [hunk 1]
```diff
@@ -67,7 +67,16 @@ message GetPointCloudMapResponse {
...
+  //
+  // Read more about the pointcloud format here:
   // https://pointclouds.org/documentation/tutorials/pcd_file_format.html
+  //
+  // Viam expects pointcloud data with fields "x y z" or "x y z rgb", and for
+  // this to be specified in the pointcloud header in the FIELDS entry. If color
+  // data is included in the pointcloud, Viam's services assume that the color
+  // value encodes a confidence score for that data point. Viam expects the
+  // confidence score to be encoded in the blue parameter of the RGB value, on a
+  // scale from 1-100.
```
Comment (JohnN193) [inline proto/viam/service/slam/v1/slam.proto:79]: Only question I have is should this description also talk about coordinate system assumptions for maps, but I think if we want to add that here then it should be a part of the documentation changes @jeremyrhyde will make for the coordinate change scope

**Review by JohnN193 — APPROVED** [review]
LGTM. Only thought is if coordinate assumptions should be documented here, but I think its fine to leave it out for this PR

File: proto/viam/service/slam/v1/slam.proto (hunk 1 above)
Comment (dmhilly) [reply proto/viam/service/slam/v1/slam.proto:79]: Good point, I'd need some more context on our coordinate system assumptions, but from what I understand that's going to change with Jeremy's work, right? If so, I think we should wait until that's resolved before documenting.

Comment (EshaMaharishi) [reply proto/viam/service/slam/v1/slam.proto:79]: Agree, I think it's okay to just document this for now.

File: component/encoder/v1/encoder.pb.gw.go [hunk 2]
```diff
@@ -15,7 +15,7 @@ import (
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
 	"github.com/grpc-ecosystem/grpc-gateway/v2/utilities"
-	"go.viam.com/api/common/v1"
```
Comment (EshaMaharishi) [inline component/encoder/v1/encoder.pb.gw.go:18]: @JohnN193 or @jeremyrhyde I know we discussed why these unrelated files get changed when we change slam.proto in https://github.com/viamrobotics/api/pull/231/files#diff-2170dff6c9599d02586a135ae17ca0a4d9e2df172c7ab24ef4a04829aad8a101. Do you remember why and mind reminding me?

**Review by EshaMaharishi — APPROVED** [review]
LGTM assuming it's correct for the .pb.gw.go files to change!

**Review by nicksanford — APPROVED** [review]
